### PR TITLE
chore/catalog: add schema test

### PR DESCRIPTION
**Pull request by caicancai**

### CODE REVIEW

**Comment by caicancai** [comment]
SchemaCatalog
```rust
pub fn new(id: SchemaId, name: String) -> SchemaCatalog {
        SchemaCatalog {
            id,
            name,
            table_idxs: HashMap::new(),
            tables: HashMap::new(),
            next_table_id: 0,
        }
    }
```
TableCatalog
```rust
pub fn new(
        id: TableId,
        name: String,
        columns: Vec<ColumnCatalog>,
        is_materialized_view: bool,
        ordered_pk_ids: Vec<ColumnId>,
    ) -> TableCatalog {
        let mut table_catalog = TableCatalog {
            id,
            name,
            column_idxs: HashMap::new(),
            columns: BTreeMap::new(),
            is_materialized_view,
            next_column_id: 0,
            ordered_pk_ids,
        };
        table_catalog
            .add_column(ColumnCatalog::new(
                u32::MAX,
                DataTypeKind::Int64.not_null().to_column("_rowid_".into()),
            ))
            .unwrap();
        for col_catalog in columns {
            table_catalog.add_column(col_catalog).unwrap();
        }

        table_catalog
    }
```

**Comment by caicancai** [comment]
I'll continue to refine this test tomorrow

**Comment by caicancai** [comment]
@wangrunji0408 hi, excuse me, if you have time, could you review my two pr? I can close this pr if there is a problem

File: src/catalog/schema.rs [hunk 1]
```diff
@@ -82,3 +82,31 @@ impl SchemaCatalog {
...
+
+        // schema
+        let mut schema_catalog = SchemaCatalog::new(0, "test".into());
+        assert_eq!(schema_catalog.id(), 0);
+        assert_eq!(schema_catalog.name(), "test");
+
+        let schema_result = schema_catalog
+            .add_table("t".into(), col_catalogs, false, vec![])
+            .unwrap();
+        assert_eq!(schema_result, 0);
```
Comment (wangrunji0408) [inline src/catalog/schema.rs:105]:
```suggestion
        let table_id = schema_catalog
            .add_table("t".into(), col_catalogs, false, vec![])
            .unwrap();
        assert_eq!(table_id, 0);
```

**Review by wangrunji0408 — APPROVED** [review]
LGTM! Thanks for your contribution.

**Comment by caicancai** [comment]
> LGTM! Thanks for your contribution.

Thank you for your review. Recently, I am studying this project to see if I can make some contributions

**Pull request merged**